sdk/communication/Azure.Communication.CallAutomation/src/Models/TransferToParticipantOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary>
+    /// The transfer call to participant operation options.
+    /// </summary>
+    public class TransferToParticipantOptions: RepeatabilityHeaders
+    {
+        /// <summary>
+        /// Creates a new TransferToParticipantOptions object.
+        /// </summary>
+        /// <param name="targetParticipant"></param>
+        /// <param name="repeatabilityRequestId"></param>
+        /// <param name="repeatablityFirstSent"></param>
+        public TransferToParticipantOptions(CommunicationIdentifier targetParticipant, Guid? repeatabilityRequestId = null, string repeatablityFirstSent = default)
```

Comment:
Needs update here also for options in constructor

Reply:
ty for catching! updated!